app/views/accounts/_pii.html.erb

```diff
@@ -4,8 +4,13 @@
       <div class="usa-alert__body">
         <div class="usa-alert__text">
           <%= t('account.re_verify.banner') %>
-
-          <%= link_to(t('account.re_verify.footer'), user_password_confirm_path) %>
+          <%= render ButtonComponent.new(
+              action: ->(**tag_options, &block) do
+                button_to(reauthentication_path, **tag_options, &block)
```

Comment:
This will need to be feature flagged as well since the route won't exist on old versions